external-crates/move/crates/move-compiler/tests/linter/suppressed_incorrect_constant_naming.move

```diff
@@ -0,0 +1,15 @@
+#[allow(lint(constant_naming))]
```

Comment:
Nice! 

Since there is going to be a lot of these lints coming, let's have one test that has all of the future lint allows in them. 

So something like 
`move-compiler/tests/linter/suppressed_lints.move`

And we can cut this test down to one case per lint, so 
```
module 0x42::M {
    const Another_BadName: u64 = 42; // Should trigger a warning
}
```

Reply:
refactored